tests/integration/test_feedback.py

```diff
@@ -139,3 +155,89 @@ def test_feedback_events_without_processing(
 
     userfeedback = envelope.items[0]
     assert userfeedback.type == "feedback"
+
+
+@pytest.mark.parametrize("use_feedback_topic", (False, True))
+def test_feedback_with_attachment_in_same_envelope(
+    mini_sentry,
+    relay_with_processing,
+    feedback_consumer,
+    events_consumer,
+    attachments_consumer,
+    use_feedback_topic,
+):
+    mini_sentry.add_basic_project_config(
+        42, extra={"config": {"features": ["organizations:user-feedback-ingest"]}}
+    )
+    mini_sentry.set_global_config_option(
+        "feedback.ingest-topic.rollout-rate", 1.0 if use_feedback_topic else 0.0
+    )
```

Comment:
This option is overwritten on either line 179 or 183, let's remove this one.

Reply:
good catch!